03-transactions.md

```diff
@@ -117,11 +117,51 @@ If a revoked commitment transaction is published, the other party can spend this
 
 #### `to_remote` Output
 
-This output sends funds to the other peer and thus is a simple P2WPKH to `remotepubkey`.
+This output sends funds to the other peer, thus is not encumbered by a
+revocation private key.
+
+If `option_anchor_outputs` applies to the commitment transaction, the `to_remote` output is delayed similarly to the `to_local` output, and is to a fixed key:
+
+    `max_to_self_delay`
```

Comment:
See comment above w.r.t switching this to something more along the lines of: `remote_to_self_delay`.

Reply:
Noted. Resolving this comment for overview.